demo/js/application.js

```diff
@@ -0,0 +1,69 @@
+
+const TestFacade = require('./testFacade');
+const Library = require('./library');
+const Printer = require('./printer');
+let readline = require('readline-sync');
+
+class Application {
+
+    test(){
```

Comment:
Is this function necessary? It seems mostly to be there as leftover code. Otherwise, what's the point of it?